src/pages/signIn/searchPage/components/Templates/SearchedRecipesList/SearchedRecipesList.jsx

```diff
@@ -0,0 +1,89 @@
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+
+import styles from './SearchedRecipesList.module.css';
+
+//import { Pagination } from '#components/Templates/Pagination/Pagination.jsx';
```

Comment:
Pagination will be implemented by backend

Reply:
@krystianklimczak 
Depends on what you mean. Backend will send the amount of recipes that you need to show on the screen plus the amount of all pages but it's frontend part to use page amount and to send to backend which page is current, etc.

Reply:
True.
Only component with state will be on front ✌️

Reply:
In relation to the above, the component remains in the same place.

Reply:
Pagination will be reusable so please implement in components folder not a page folder 💡